pov_tools/create_POV_data_files

```diff
@@ -0,0 +1,218 @@
+            c)  calc_len=$(printf %.0f "$((10**3 * $height * 100 / 144))e-3"); offset_default=46; validinput=1; units="centimeters"; echo "";;
+            *[q*]*) exit;;
+            *)  echo ""
+				echo "! Not a valid option, Please choose inches or cm,"
+				echo "or type q to quit. "; echo "";;
+        esac
+    done
+
+	validinput=0
+	while [[ "$validinput" != 1 ]] && [[ $preset == 0 ]]
+	do
+		echo "How many "$units" is your blade? (Press ENTER for calculated default $calc_len) "
+		read length
+		if [[ $units == "inches" ]]; then max=48; else max=122; fi
+		if [[ $length == "" ]]; then length=$calc_len validinput=1; echo "Default $calc_len"
+		elif [[ $length == "q" ]]; then exit
+		elif [[ $length -gt $max ]]; then
+			read -p "Are you sure your blade is that long?? (y/n) " too_long
+			if [[ $too_long == "y" ]]; then validinput=1; else echo ""; fi	
+		elif [[ $length =~ ^[0-9]{1,3}$ ]]; then validinput=1
+		else
+			echo "! That is likely impossible."
+			echo "  Please try again or type 'q' to quit."
+			echo ""
+		fi
+	done
+
+	validinput=0
+	while [[ "$validinput" != 1 ]] && [[ $preset == 0 ]]
+	do
+        echo ""
+        echo "How many "$units" is the distance from the bottom of the blade to the fulcrum?"
+        echo "(The fulcrum is usually the swinging joint) (Press ENTER for default $offset_default)"
+		read offset
+		if [[ $units == "inches" ]]; then max=30; else max=76; fi
+		if [[ $offset == "" ]]; then  offset=$offset_default validinput=1; echo "Default $offset_default"
+		elif [[ $offset == "q" ]]; then exit
+		elif [[ $offset -gt $max ]]; then
+			read -p "Are you sure this distance is that long?? (y/n) " long_arm
+			if [[ $long_arm == "y" ]]; then validinput=1; else echo ""; fi	
+		elif [[ $offset =~ ^[0-9]{1,3}$ ]]; then validinput=1
+		else
+			echo "! That is likely impossible."
+			echo "  Please try again or type 'q' to quit."
+			echo ""
+		fi
+	done
+else # height_only
+	length=39 offset=18
+fi
+
+if [ $preset == 0 ]; then make IMAGE=$image OPTIONS="--height=$height  --length=$length --offset=$offset"; HEIGHT=$height; fi
+mv ./preview.png ./"${sourcebase}"_"${HEIGHT}"_preview.png
+mv ./image.h ../styles/"${sourcebase}"_"${HEIGHT}"_FC_POV_data.h
+mv ./image_pgm.h ../styles/"${sourcebase}"_"${HEIGHT}"_SC_POV_data.h
+mv ./image_8bit.h ../styles/"${sourcebase}"_"${HEIGHT}"_8b_POV_data.h
+echo ""
+echo ""
+echo "Conversion DONE! Find your files as shown below."
+echo ""
+echo "preview image:"
+echo "	ProffieOS/pov_tools/"$sourcebase"_"$HEIGHT"_preview.png"
+echo "Full color image data file:"
+echo "	ProffieOS/styles/"$sourcebase"_"$HEIGHT"_FC_POV_data.h"
+echo "Single color image data file:"
+echo "	ProffieOS/styles/"$sourcebase"_"$HEIGHT"_SC_POV_data.h"
+echo "256 color image data file:"
+echo "	ProffieOS/styles/"$sourcebase"_"$HEIGHT"_8b_POV_data.h"
+rm pnmtorle; rm pnmquantizedtorle; rm pgmtorle; rm pnmwindshieldwiper
```

Comment:
What's the point of deleting these?

Reply:
Just cleaning up after the process is all. I had asked about these hex versions being generated and you didn't know why, which to me means they're superfluous once the job is done.  They could just as well stay if it's a problem removing them.

Reply:
I think I misunderstood then.
These files are the actual programs. If you delete them they are going to need to be re-compiled each time which slows things down.